docs/tutorialkit.dev/src/content/docs/reference/theming.mdx

```diff
@@ -48,12 +48,22 @@ These tokens can be used to style the colors of the tutorial on a high level. Fo
 | `--tk-elements-app-textColor`           | The text color of the TutorialKit app.       |
 | `--tk-elements-app-backgroundColor`     | The background color of the TutorialKit app. |
 | `--tk-elements-app-borderColor`         | The border color of the TutorialKit app.     |
-| `--tk-elements-app-linkColor`           | The link color of the TutorialKit app.       |
 
 If you want to start with a `theme.css` file that includes all the available CSS variables you can use the following snippet:
 
 <Code code={themeCSS} lang="css" title="theme.css" class="limit-code-height" />
 
+### Links
+
+Tokens used by links.
+
+| Token                                    | Description                                          |
+| ---------------------------------------- | ---------------------------------------------------- |
+| `--tk-elements-link-primaryColor`        | The text color of the primary links.                 |
+| `--tk-elements-link-primaryHoverColor`   | The text color of the primary links when hovering.   |
```

Comment:
Sorry another nit 🙈 : `-primaryHoverColor` -> `-primaryColorHover`

Reply:
Oh good catch - this is supposed to be `primaryColorHover` 🙃 
Need to change in the code as well